hbase-client/src/main/java/org/apache/hadoop/hbase/client/OnlineLogRecord.java

```diff
@@ -330,7 +373,14 @@ public String toString() {
       .append("methodName", methodName).append("callDetails", callDetails).append("param", param)
       .append("regionName", regionName).append("userName", userName)
       .append("multiGetsCount", multiGetsCount).append("multiMutationsCount", multiMutationsCount)
-      .append("multiServiceCalls", multiServiceCalls).append("scan", scan).toString();
+      .append("multiServiceCalls", multiServiceCalls).append("scan", scan)
+      .append("requestAttributes", requestAttributes)
+      .append("connectionAttributes", connectionAttributes).toString();
+  }
+
+  public static Map<String, String> deserializeAttributes(Map<String, byte[]> attributes) {
+    return attributes.entrySet().stream()
+      .collect(Collectors.toMap(Map.Entry::getKey, entry -> Bytes.toString(entry.getValue())));
   }
```

Comment:
I made this public because we'll want to reuse this logic when deserializing attributes for the RS UI's Operation Details page